### ISSUE TITLE
#[target_feature] form error

### ISSUE THREAD

**Issue by yhql** [rust-lang-nursery/stdsimd#489]
Hi, cannot compile on windows x86, rust-nightly 1.28 due to (973 instances of) this : 

``` 
         |
2740 | #[target_feature = "+avx2"]
         | ^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: #[target_feature] attribute must be of the form #[target_feature(..)]
```
coresimd is used through the `faster` crate here. Did I miss something ?

**Comment by yhql** [comment]
6 second late duplicate of https://github.com/rust-lang-nursery/stdsimd/issues/488

**Comment by gnzlbg** [comment]
@yhql see the fix in #488 - the problem is that `faster` is using `stdsimd` from `cargo` which is deprecated since `stdsimd` became a part of std. Please feel free to open an issue on `faster` upstream.

**Comment by yhql** [comment]
Thank you @gnzlbg :) will do
